test/conformance/chainsaw/generate/clusterpolicy/cornercases/cpol-data-trigger-not-present/chainsaw-test.yaml

```diff
@@ -46,5 +46,8 @@ spec:
   - name: step-07
     try:
     - script:
-        content: "if kubectl get updaterequests -n kyverno 2>&1 | grep -q 'No resources found in kyverno namespace.'\nthen \n  exit 0 \nelse \n  exit
-          1\nfi\n"
+        content: kubectl get updaterequests -n kyverno
```

Comment:
For the record, this kind of test can easily be implemented with an `error` statement.

Reply:
Something like this?

```
apiVersion: chainsaw.kyverno.io/v1alpha1
kind: TestStep
metadata:
  name: example
spec:
  try:
  # ...
  - error:
      resource:
        apiVersion: v1
        kind: Deployment
        metadata:
          name: foo
        spec:
          (replicas > 3): true
  # ...
```

Reply:
Something like that
```yaml
  - error:
      resource:
        apiVersion: <put the right apiVersion here>
        kind: UpdateRequest
        metadata:
          namespace: kyverno
```

Reply:
If there is any it will be considered an error